tests/permissions_tests/models.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
```

Comment:
I think it would be better to put these tests in `tests/auth_tests` unless there's some reason that's not feasible.